src/test/java/org/folio/test/support/ApiTests.java

```diff
@@ -155,13 +156,25 @@ protected OkapiDeployment getOkapi() {
 
   protected void removeAllFromTable(String tableName) {
     final CompletableFuture<Void> future = new CompletableFuture<>();
-
+    Criterion criterion = new Criterion();
+    if ("feefines".equals(tableName)) {
```

Comment:
What if we move the constants with table names from FeeFinesAPITest to ApiTests and use it here